lib/experimental/class-custom-wp-posts-list-table.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Custom posts list table class to add avatar indicators for posts with open block comments.
+ *
+ * Extends the core class with only one line changes in the column_title method.
+ */
+class Custom_WP_Posts_List_Table extends WP_Posts_List_Table {
+	// Avatar colors matching client side code in https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/collab-sidebar/utils.js#L20-L29
```

Comment:
```suggestion
	// Avatar colors matching client side code in https://github.com/WordPress/gutenberg/blob/41e1fe574ecc5c6719d25f77db3a314825c914c4/packages/editor/src/components/collab-sidebar/utils.js#L20-L29
```

Let's use the commit hash since the lines may be out of sync.